### PR TITLE
BEM-36: Add my contents checkbox

### PR DESCRIPTION
**Pull request by paolosantarsiero**
Add a checkbox in modules views to filter items created and modified for last time by current user.

### CODE REVIEW

File: src/Template/Layout/js/app/app.js [hunk 1]
```diff
@@ -176,6 +176,11 @@ const _vueInstance = new Vue({
          * @return {void}
          */
         onFilterObjects(filter) {
+            // remove from query string ilter mine if it is false
+            if (!filter.filter.mine) {
```
Comment (stefanorosanelli) [inline src/Template/Layout/js/app/app.js]: why do we need this?

**Review by stefanorosanelli — COMMENTED** [review]
Seems ok

On default filters the checkbox is too close to the `Reset Filters` button, see below

image

File: src/Template/Layout/js/app/app.js (hunk 1 above)
Comment (paolosantarsiero) [reply src/Template/Layout/js/app/app.js]: I need this to remove `filter.mine` from `url` (in query string) when it is false and unchecked

Comment (paolosantarsiero) [reply src/Template/Layout/js/app/app.js]: This happened because when `filter.mine` is `false` or `0` it seems like enabled.

**Review by stefanorosanelli — APPROVED** [review]

**Comment by stefanorosanelli** [comment]
currently we have only `mine` filter => objects created by the logged user
it's fine for me now

**Review by didoda — APPROVED** [review]

**Review by didoda — APPROVED** [review]

**Pull request merged**